esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -431,7 +432,8 @@ void EsdlServiceImpl::configureUrlMethod(const char *method, IPropertyTree &entr
     entry.setProp("@prot", protocol);
     entry.setProp("@path", path);
 
-    Owned<ISmartSocketFactory> sf = createSmartSocketFactory(iplist, true);
+    Owned<ISmartSocketFactory> sf = new CSmartSocketFactory(iplist, true);
```

Comment:
The create...() does give you an instance of SmartSocket. We use many create...() functions to create a class instance. The create...() functions may 'encapsulate' some pre-processes with the instance creation. It looks wired to change from an create...() to new...() although the code does work...